docs/sql-ref-functions-udf-aggregate.md

```diff
@@ -94,8 +94,45 @@ For example, a user-defined average for untyped DataFrames can look like:
 <div data-lang="java"  markdown="1">
   {% include_example untyped_custom_aggregation java/org/apache/spark/examples/sql/JavaUserDefinedUntypedAggregation.java%}
 </div>
+<div data-lang="sql"  markdown="1">
```

Comment:
`sql` -> `SQL`

Reply:
Thanks for your review.
I did a quick check, seems all the other places use lower case ``sql```, so I will use lower case for  consistency.
<img width="459" alt="Screen Shot 2020-04-13 at 5 42 28 PM" src="https://user-images.githubusercontent.com/13592258/79173998-324e5f80-7dae-11ea-9e78-c20260466256.png">

Reply:
Ah, I see. The uppercase looks better to me. How about fixing them in minor follow-up?

Reply:
Sure. I will fix them all in this PR.